### DISCUSSION TITLE
How the forces are calculated in 2D Simulation?

### DISCUSSION THREAD

**Discussion by kursatyurt** [su2code/SU2#2037] [General]
Hi, 

I have 2D airfoil simulation which I am coupling with a 2D structure. As a part of modeling, I have to introduce some length in the z-direction to the structural model to calculate some stiffness values. One of the reference papers uses a width of 0.1m and the other one takes a 1m span in z-direction.

Is there any way to specify the reference depth in 2D simulations? If not what is the reference value?

Best,
Kursat Yurt

**Comment by pcarruscag** [comment]
Hi, in 2D the forces are per unit lenght, you can multiply them by the depth you want or assume 1m.

**Reply by kursatyurt** [reply]
Thank you for the clarification.
